### PR TITLE
World cup

### PR DESCRIPTION
**Pull request by tmaria17**
Maria Torres

### CODE REVIEW

File: lib/player.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Player
+  attr_reader :name,
+              :position
+
+  def initialize(name,position)
+    @name = name
+    @position = position
```
Comment (s-espinosa) [inline lib/player.rb]: 👍 Looks good!

File: lib/player.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+
```
Comment (s-espinosa) [inline lib/player.rb]: Be sure to delete this whitespace as you finish working in your class.

File: lib/team.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require './lib/player.rb'
+class Team
+  attr_reader :country,
+              :players,
+              :player
```
Comment (s-espinosa) [inline lib/team.rb]: Looks like you're missing a comma here. May throw off the use of the `attr_reader`s.

File: lib/team.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+              :eliminated
+
+
+  def initialize(country, eliminated = false)
+    @country = country
+    @players = [ ]
+  @eliminated = eliminated
```
Comment (s-espinosa) [inline lib/team.rb]: 👍

Watch your indentation!

File: lib/team.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+
+  end
+  def eliminated?
+    @eliminated
+  end
```
Comment (s-espinosa) [inline lib/team.rb]: 🎉 Like this!

File: lib/team.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
+require './lib/player.rb'
+class Team
+  attr_reader :country,
+              :players,
+              :player
+              :eliminated
```
Comment (s-espinosa) [inline lib/team.rb]: Think this needs to be an `attr_writer` to allow users to access this value from outside the class.

File: lib/team.rb [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  @eliminated = eliminated
+
+
+  end
+  def eliminated?
+    @eliminated
+  end
+
+  def add_player(player)
+    @players.push(player)
```
Comment (s-espinosa) [inline lib/team.rb]: Yup!

File: lib/team.rb [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  end
+
+def players_by_position(type)
+  @players.find_all do |player|
+     player.position == type
+
+   end
```
Comment (s-espinosa) [inline lib/team.rb]: Nice and concise!

File: test/player_test.rb [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    assert_equal "Paul Pogba" , pogba.name
+  end
+
+  def test_player_has_position
+    pogba = Player.new("Paul Pogba", :midfielder)
+
+    assert_equal :midfielder , pogba.position
+
+
+  end
```
Comment (s-espinosa) [inline test/player_test.rb]: Assertions look good! Clean up the whitespace and this is in good shape.

File: test/team_test.rb [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+    assert_equal [], france.players
+
+  end
+
+  def test_elminated_default_is_false
+    france = Team.new("France")
+
+    assert_equal false , france.eliminated?
+
```
Comment (s-espinosa) [inline test/team_test.rb]: 🎉

File: test/team_test.rb [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  def test_that_team_can_be_eliminated
+    france = Team.new("France",eliminated = true)
```
Comment (s-espinosa) [inline test/team_test.rb]: Careful here! The interaction pattern doesn't allow you to pass `eliminated` as an argument.

Also, while this works, I don't think that it's doing what you think it's doing? Using `eliminated = true` *where you define the method* will allow you to set a default value for a parameter. Passing it `eliminated = true` where you call the method (here in your test) is really passing the *return value* of `eliminated = true`, which happens to be `true` because assignment returns the value on the right of the equal sign.

In short, don't use an `=` in the parenthesis where you call your method.
